### PR TITLE
constants, navbar, bg header, Video Image, colors...

### PR DESCRIPTION
**Pull request by DrielisonLopes**

### CODE REVIEW

File: env.local [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Configurações do Firebase
```
Comment (renanlopescoder) [inline env.local:1]: Firebase config should be in constants, it's fine for security as they can access it anyway

File: lib/firebase.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// @ts-nocheck
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+
+const firebaseConfig = {
+  apiKey: FIREBASE_API_KEY,
```
Comment (renanlopescoder) [inline lib/firebase.ts:6]: The firebase config can go directly here

File: src/components/molecules/ImageCarousel/ImageCarousel.tsx [hunk 3]
```diff
@@ -20,90 +22,96 @@ const ImageCarousel = ({ images }) => {
...
-        Enact malign schemes and trickery with the Changeling of Tzeentch.
-        Establish and execute grand stratagems on a global scale as Yuan Bo of Grand Cathay.
-        Conjure powerful hexes and incantations whilst engaging in witchcraft, flipping campaigns on their head as Mother Ostankya of Kislev. 
-        Scatter magical spells far and wide with the Blue Scribes of Tzeentch; a new Legendary Hero. 
-        ​Expand your roster with 12 additional overall units, and enhance your troops with 9 new Regiments of Renown.
-      </p>
+      <div className={styles.gameDescription}>
+        <h1>About City Tank</h1>
+        <p>
+          ​Inject new Legendary Lords, Heroes, and Units into your games to vastly expand your methods of play on the campaign map and within battles. The Changeling, Yuan Bo, and Mother Ostankya each bring their own unique campaign mechanics to their races, ​ ​with goals separate from the Ursus storyline in the Realm of Chaos campaign, and new units to help them pursue victory on their terms.
```
Comment (renanlopescoder) [inline src/components/molecules/ImageCarousel/ImageCarousel.tsx:105]: This content should be inside the constant of the specific game instead here for all

File: src/constants/index.ts [hunk 4]
```diff
@@ -12,15 +12,33 @@ export const GAMES: Game[] = [
     externalLink:
```
Comment (renanlopescoder) [inline src/constants/index.ts:12]: City Tank should be added somewhere here, maybe a new list of games like PIXEL_FORGE_GAMES ?

File: env.local (hunk 1 above)
Comment (DrielisonLopes) [reply env.local:1]: The idea was to put env.local in git ignore so it wouldn't appear here.

File: src/components/molecules/ImageCarousel/ImageCarousel.tsx (hunk 3 above)
Comment (DrielisonLopes) [reply src/components/molecules/ImageCarousel/ImageCarousel.tsx:105]: is adjusted in the next PR

File: src/constants/index.ts (hunk 4 above)
Comment (DrielisonLopes) [reply src/constants/index.ts:12]: created ;)

File: lib/firebase.ts (hunk 2 above)
Comment (DrielisonLopes) [reply lib/firebase.ts:6]: but wouldn't it be unprotected by placing it directly here?

Comment (renanlopescoder) [reply lib/firebase.ts:6]: It's fine this info can be public

**Review by renanlopescoder — APPROVED** [review]

**Pull request merged**